hugr-core/src/extension/op_def.rs

```diff
@@ -186,15 +186,21 @@ impl<T: CustomSignatureFunc + 'static> From<T> for SignatureFunc {
     }
 }
 
-impl From<PolyFuncType> for SignatureFunc {
-    fn from(v: PolyFuncType) -> Self {
+impl From<TypeSchemeRV> for SignatureFunc {
```

Comment:
Rust complains, because some future type could define both Into<TypeSchemeRV> (currently `Into<PolyFuncTypeRV>`) and `Into<CustomSignatureFunc>` and then it would be ambiguous...

Reply:
I tried `impl <RV: MaybeRV> From<PolyFuncTypeBase<RV>> for SignatureFunc` but there is no conversion from `PolyFuncTypeBase<RV>` (**parametrized**) to `<RowVariable>`... I could add a `.into_()` but would rather avoid that if I can...

Reply:
I can add the fourth (`From<PolyFuncType>`) but I couldn't see a scenario where you're making an OpDef but didn't know to create a `PolyFuncTypeRV` (rather than `PolyFuncType`) in the first place...

Reply:
It's easy to add though, done